cluster-scope/overlays/nerc-ocp-prod/machineconfigs/configure-bond0/bond0-controller.yaml

```diff
@@ -0,0 +1,28 @@
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: configure-bond0-controller
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            compression: gzip
+            source: data:;base64,H4sIAAAAAAAC/0zMwaqDMBCF4f15FyXeKxfvYp5EskiTKQY6M6IToW9fLEK7O/BzvjmbKmevphG10M20BPhz5fdEVeftnjJ3moSvnJrb9SPfGuOD9F+t2x/p4J0GYGZfeFP2CPFG/yEEYD61CLHCNIWf/jcVSK1iSsN49roeY4SwL1bohFGWvHZeha05TX9jCHgFAAD///WlE5PCAAAA
```

Comment:
@jtriley how come bond0 file is encoded/compressed, but nic1 and nic2 are not?

Reply:
Good question - this file gets generated by butane in the `Makefile`. My guess is that is governed by the size of the result. According to line count, bond0 file is larger than nic1 and nic2:

```
$ wc -l src/*.nmconnection
      17 src/bond0.nmconnection
      10 src/nic1.nmconnection
      10 src/nic2.nmconnection
```